icubam/db/sqlite.py

```diff
@@ -143,11 +153,11 @@ def get_icus(self):
   def get_users(self):
     """Returns a pandas DF of bed counts."""
     return pd.read_sql_query(
-      """SELECT users.icu_id, icu_name, name, users.telephone, description
-         FROM users
-         JOIN icus
-         ON users.icu_id = icus.icu_id""",
-      self._conn,
+        """SELECT users.icu_id, icu_name, name, users.telephone, description
+           FROM users
+           JOIN icus
+           ON users.icu_id = icus.icu_id""",
+        self._conn,
     )
 
   def get_bedcount(self, icu_ids: Sequence = None, max_ts=None):
```

Comment:
IS this something we can do in SQLAlchemy?  This was the piece of code that originally made me want to stop writing queries by hand.

Reply:
Sure, converted it to a more sane form. Please see the note regarding the update_ts.